components/camel-aws/camel-aws2-kinesis/src/main/java/org/apache/camel/component/aws2/kinesis/Kinesis2Configuration.java

```diff
@@ -52,6 +52,9 @@ public class Kinesis2Configuration implements Cloneable {
     @UriParam(label = "consumer",
               description = "The sequence number to start polling from. Required if iteratorType is set to AFTER_SEQUENCE_NUMBER or AT_SEQUENCE_NUMBER")
     private String sequenceNumber = "";
+    @UriParam(label = "consumer",
+            description = "The message timestamp to start polling from. Required if iteratorType is set to AFTER_SEQUENCE_NUMBER or AT_SEQUENCE_NUMBER")
```

Comment:
required if iteratorType is AT_TIMESTAMP ?
is it really required ?

Reply:
For AT_TIMESTAMP only. I'll fix this description, thank you